src/ansys/dpf/core/_version.py

```diff
@@ -1,6 +1,6 @@
 """Version for ansys-dpf-core"""
 # major, minor, patch
-version_info = 0, 8, 1, "dev0"
+version_info = 0, 9, 0, "dev0"
```

Comment:
@PProfizi is that ok?

Reply:
@anslpa, sure, meaning that our pydpf-core libraries developed for 2024R1 will be at least 0.9.0.